src/axom/inlet/Field.cpp

```diff
@@ -64,11 +64,14 @@ bool Field::required()
   return (bool)intValue;
 }
 
-std::shared_ptr<Field> Field::addDefaultString(std::string value) {
-  SLIC_ASSERT_MSG(m_type == FieldType::STRING, "Default value type did not match STRING");
+std::shared_ptr<Field> Field::addDefaultValue(const char* value) {
```

Comment:
I'd lean towards not giving them this functionality. But if you want it you need to check for nullptr before converting to string because you will get a runtime error.